migration/v2.26/README.md

```diff
@@ -90,6 +92,32 @@ These steps will cause disruptions in the environment.
         ]
     ```
 
+    Else if `loadbalancer_proxy_protocol = true` is not present in the cluster.tfvars file, do the following:
```

Comment:
Or should we encourage to enable proxy protocol, but also note that you must set the ingress-nginx controller to proxy protocol mode? Since we want to be able to see source IPs in cluster

Reply:
Then that need to have a caution notice since it _will_ be highly disruptive during the switch, and one should notify the application developers.

Reply:
Very true, then I agree that we should keep as is